docs/getstarted/userinterface.md

```diff
@@ -279,6 +279,36 @@ You can Drag and Drop editor groups on the workbench, move individual Tabs betwe
 
 >**Note:** VS Code uses editor groups whether or not you have enabled Tabs.  Without Tabs, editor groups are a stack of your open items with the most recently selected item visible in the editor pane.
 
+### Locked editor groups
+
+By default, files will always open in the active editor group. Locked editor groups help to avoid this. The `workbench.editor.autoLockGroups` setting allows you to select editors that should lock a group automatically when they open. This only applies when the editor is the first to open in an otherwise empty or new group.
+
+```json
+"workbench.editor.autoLockGroups": {
+  "default": false,
+  "terminalEditor": true
+}
+```
+
+Terminals are configured by default to cause a new group to lock automatically. A lock icon in the action toolbar (top right) indicates a locked group. The locked state of an editor group is persisted and restored across restarts. You can lock empty groups for a more stable editor layout.
+
+<!-- Required? Locked editor https://code.visualstudio.com/assets/updates/1_61/editor-readonly-deleted.png -->
+
+You can control the behavior of locked editor groups with these settings:
```

Comment:
Aren't these command ids rather than settings?

Reply:
No idea, the [release notes documents linked in this issue](https://github.com/microsoft/vscode-docs/issues/6312) said settings, see also this line: https://github.com/microsoft/vscode-docs/pull/6337/files#diff-cd59c6f34706e30277502d5e00f8379b8449e9e91b879ca315da48e76ba42f17R284

Atm, I have no clue, to be honest.